### DISCUSSION TITLE
简单二次开发

### DISCUSSION THREAD

**Discussion by SiQuan-Wen** [YunYouJun/hexo-theme-starry#162] [Show 分享展示]
name: 小文的博客
url: https://siquan-wen.github.io/
avatar: https://github.com/SiQuan-Wen/SiQuan-Wen.github.io/blob/hexo/source/images/avatar.jpg
desc: 救赎之道，就在其中。

**Comment by zhoubin-me** [comment]
二次开发？新添了啥功能？

**Reply by YunYouJun** [reply]
不知道（）

**Reply by SiQuan-Wen** [reply]
呃呃呃，就是改了UI，吼吼。
